DataFormats/Detectors/ITSMFT/common/src/ClusterPattern.cxx

```diff
@@ -29,7 +29,7 @@ ClusterPattern::ClusterPattern(int nRow, int nCol, const unsigned char patt[Clus
   setPattern(nRow, nCol, patt);
 }
 
-ClusterPattern::ClusterPattern(std::vector<unsigned char>::iterator& pattIt)
+ClusterPattern::ClusterPattern(std::vector<unsigned char>::const_iterator& pattIt)
```

Comment:
See comment below, why not using simply ``ClusterPattern(const unsigned char*& patt)`` ?

Reply:
This method is used in CheckCOG.C, for example, to access the elements of the pattern branch. But I agree with @shahor02, it can be changed in this way and than it is possible to work on the content of the iterator.

Reply:
@shahor02 @lbariogl See my comment above.